exe/rubocop

```diff
@@ -9,6 +9,18 @@ require 'benchmark'
 cli = RuboCop::CLI.new
 result = 0
 
+if defined?(Bundler)
+  gemfile_lock = Bundler.read_file(Bundler.default_lockfile)
+  parser = Bundler::LockfileParser.new(gemfile_lock)
+
+  unless parser.dependencies['rubocop-performance']
+    warn Rainbow(<<-MESSAGE.strip_indent).yellow.to_s
```

Comment:
📝 The above `.to_s` method is used to prevent the following errors for JRuby 9.2.

```console
% bundle exec rspec ./spec/rubocop/cli/cli_options_spec.rb:210
Run options: include {:focus=>true, :locations=>{"./spec/rubocop/cli/cli_options_spec.rb"=>[210]}}

Randomized with seed 11663
TypeError: wrong argument type Rainbow::NullPresenter (expected String)
    warn at org/jruby/common/RubyWarnings.java:224
    warn at org/jruby/RubyKernel.java:1247
    warn at org/jruby/RubyKernel.java:1234
  <main> at /Users/koic/src/github.com/rubocop-hq/rubocop/exe/rubocop:17
F

Failures:

  1) RuboCop::CLI --only when one cop is given accepts cop names from plugins
     Got 2 failures from failure aggregation block.
     # ./spec/support/cli_spec_behavior.rb:27:in `block in <main>'
     # ./lib/rubocop/rspec/shared_contexts.rb:27:in `block in <main>'
     # ./lib/rubocop/path_util.rb:67:in `chdir'
     # ./lib/rubocop/rspec/shared_contexts.rb:26:in `block in <main>'
     # ./lib/rubocop/rspec/shared_contexts.rb:8:in `block in <main>'

     1.1) Failure/Error: expect($CHILD_STATUS.success?).to be_truthy

            expected: truthy value
                 got: false
          # ./spec/rubocop/cli/cli_options_spec.rb:210:in `block in <main>'

     1.2) Failure/Error:
                    expect(output)
                      .to eq(<<-RESULT.strip_indent)
                        Inspecting 2 files
                        ..

                        2 files inspected, no offenses detected
                      RESULT
```

https://circleci.com/gh/rubocop-hq/rubocop/42485